pkg/cmd/clientConfig.go

```diff
@@ -16,24 +16,33 @@ package cmd
 
 import (
 	"encoding/json"
-	"os"
+	"fmt"
+	"io"
 
+	"github.com/aerogear/mobile-cli/pkg/cmd/output"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"k8s.io/client-go/kubernetes"
 )
 
+// ClientConfigCmd executes the retrieval and display of the client config
 type ClientConfigCmd struct {
+	*BaseCmd
 	k8Client kubernetes.Interface
 }
 
-func NewClientConfigCmd(k8Client kubernetes.Interface) *ClientConfigCmd {
+// NewClientConfigCmd creates and returns a ClientConfigCmd object
+func NewClientConfigCmd(k8Client kubernetes.Interface, out io.Writer) *ClientConfigCmd {
 	return &ClientConfigCmd{
 		k8Client: k8Client,
+		BaseCmd:  &BaseCmd{Out: output.NewRenderer(out)},
 	}
 }
 
+// GetClientConfigCmd returns a cobra command object for getting client configs
 func (ccc *ClientConfigCmd) GetClientConfigCmd() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "clientconfig",
```

Comment:
can we update the the command to use a similar approach as this command https://github.com/aerogear/mobile-cli/blob/master/pkg/cmd/clients.go#L86